src/test/model/graph.test.ts

```diff
@@ -141,9 +141,78 @@ describe('Graph', () => {
 
     describe('invariants', () => {
 
-        it('removes extra properties not supported by node definition');
-        it('ensures the correct order properties');
-        it('initializes all properties with default values');
+        it('keeps only firstmost properties supported by node definition', async () => {
```

Comment:
I'd still split this case into two, because it essentially does a check for two different effects of the same code that aren't necessarily related.